folding-schemes/src/utils/gadgets.rs

```diff
@@ -107,3 +107,35 @@ impl<F: PrimeField> MatrixGadget<FpVar<F>> for SparseMatrixVar<F, F, FpVar<F>> {
             .collect())
     }
 }
+
+/// Interprets the given vector v as the evaluations of a dense multilinear extension of n_vars,
+/// and evaluates it at the given point. This method mimics the behaviour of
```

Comment:
```suggestion
/// and evaluates it at the given point. This method mimics the behavior of
```

Reply:
cool, updated from british to american version

Reply:
lol what? Is this real? I didn't know. Did not want to be pedantic at all. Rather I though was incorrect. Apologies. Leave as you wish.

Reply:
haha it's fine, already updated to american, I'm happy both ways